cli/root_test.go

```diff
@@ -104,6 +105,39 @@ func TestRootCmd_SimplePromptOnly(t *testing.T) {
 	wg.Wait()
 }
 
+func TestRootCmd_SimpleClipboard(t *testing.T) {
+	prompt := "Say: Hello World!"
+	expected := "Hello World!\n"
+
+	mem := &exitMemento{}
+	var wg sync.WaitGroup
+	reader, writer := io.Pipe()
+
+	config := createTestConfig(t)
+
+	root := newRootCmd(mem.Exit, config, mockIsPipedShell(false, nil), api.MockClient(strings.Clone(expected), nil))
+	root.cmd.SetOut(writer)
+
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		var buf bytes.Buffer
+		_, err := io.Copy(&buf, reader)
+		_ = clipboard.WriteAll(buf.String())
```

Comment:
Here we are overriding the values which were set in the root command. Therefore we are not actually testing the functionality.

Reply:
Sorry you are right

Reply:
fixed